### PR TITLE
Add docs for data parameter in alert component

### PR DESCRIPTION
**Pull request by syphernl**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#17616

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/alert.markdown [hunk 1]
```diff
@@ -87,6 +92,16 @@ skip_first:
   required: false
   type: boolean
   default: false
+data:
+  description: >
+    Dictionary of extra parameters to send to the notifier.
+  required: false
+  type: dictionary
```
Comment (frenck) [inline source/_components/alert.markdown]: `dictionary` is not a valid type for the documentation.

File: source/_components/alert.markdown [hunk 2]
```diff
@@ -87,6 +92,16 @@ skip_first:
...
+data_template:
+  description: >
+    emplate dictionary of extra parameters to send to the notifier.
+  required: false
+  type: dictionary
```
Comment (frenck) [inline source/_components/alert.markdown]: `dictionary` is not a valid type for the documentation.

File: source/_components/alert.markdown [hunk 3]
```diff
@@ -192,4 +207,36 @@ following notification.
 For example, if the garage door opens at 2:00, a notification will be
 sent at 2:15, 2:45, 3:45, 4:45, etc., continuing every 60 minutes.
 
+
+### {% linkable_title Additional parameters for notifiers  %}
+
+Some notifiers support more parameters (e.g. to set text color or action
```
Comment (frenck) [inline source/_components/alert.markdown]:
```suggestion
Some notifiers support more parameters (e.g., to set text color or action
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @syphernl! I've reviewed it and left you some comments.
Could you please take a look? 👍

**Comment by syphernl** [comment]
@frenck Thanks for your feedback.I have made the requested changes.

**Comment by syphernl** [comment]
@frenck Thanks for your feedback.I have made the requested changes.

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Thanks for updating the PR @syphernl! 👍 

✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by syphernl** [comment]
This isn't stale.

File: source/_components/alert.markdown [hunk 4]
```diff
@@ -58,6 +58,11 @@ entity_id:
   description: The ID of the entity to watch.
   required: true
   type: string
+title:
+  description: >
+    A title to be used for the notification if the notifier supports it.
```
Comment (MartinHjelmare) [inline source/_components/alert.markdown]: We should mention that this can be a template.

Comment (syphernl) [reply source/_components/alert.markdown]: Done!

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Pull request merged**